src/pages/office/office.ts

```diff
@@ -198,6 +222,9 @@ export class OfficePage extends BasePageComponent {
         select(getVehicleChecks),
         select(getShowMeQuestion),
       ),
+      showMeQuestionOptions$: currentTest$.pipe(
+        map(() => getShowMeQuestionOptions(this.showMeQuestions, '4', this.outcomeBehaviourProvider)),
```

Comment:
Perhaps a TODO comment here to replace the hardcoded outcome?

Reply:
definite TODO needed. Changing now.

Reply:
Have wired this up to take outcome from state, so no TODO needed